### PR TITLE
WIP INFS-97 - lbaker/INFS-97/pg13.6

### PR DESCRIPTION
**Pull request by lbakerchef**
### Description

Upgrade to PG 13.6

### Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussions that are relevant]

### Check List

- [ ] New functionality includes tests
- [ ] All buildkite tests pass
- [ ] Full omnibus build and tests in buildkite pass
- [ ] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/main/CONTRIBUTING.md#developer-certification-of-origin-dco>
- [ ] PR title is a worthy inclusion in the CHANGELOG

### CODE REVIEW

**Comment by PrajaktaPurohit** [comment]
I think this one needs a rebase and an adhoc run.

**Review by marcparadise — COMMENTED** [review]
This looks fine so far, but would like to see the rebar.lock changes in their own PR; and I think this accidentally reverts some changes related to opensearch support.

File: omnibus/files/server-ctl-cookbooks/infra-server/libraries/preflight_indexing_validator.rb [hunk 1]
```diff
@@ -41,7 +41,6 @@ def run!
     verify_es_disabled_if_user_set_external_solr
```
[the remark this answers is not in the document]
Comment (vinay-satish) [reply omnibus/files/server-ctl-cookbooks/infra-server/libraries/preflight_indexing_validator.rb]: Fixed it.

**Comment by vinay-satish** [comment]
Adhoc build - https://buildkite.com/chef/chef-chef-server-main-omnibus-adhoc/builds/4116

**Review by PrajaktaPurohit — APPROVED** [review]

**Pull request merged**